### ISSUE TITLE
Implement temporary effects, for example poison

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#62]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.

I find myself going to the snake lair a lot for meat and thought that if the 
snakes poisioned you when they attacked that would make it more difficult and 
ofcourse that would make a need for antidote potions. The attack could take 
like 1hp every five seconds or so even after the monster was destroyed. So if 
you have no antidote potions in your inventory, better get to the nearest town 
and get some.

```

Original issue reported on code.google.com by `derrickv...@yahoo.com` on 4 Dec 2010 at 1:48

**Comment by GoogleCodeExporter** [comment]
```
There is a generalized version of this that I have though about, namely 
temporary effects. That is something that I have wanted to implement but 
haven't though out completely yet. Temporary effects could including posioning, 
combat inihibiting effects (curses?), and also positive effects such as bless 
and temporary increased combat proficiency.

We would need some way of indicating such an effect in the UI, some way of 
starting the effect and some way of applying the effect in code.

Good idea!
```

Original comment by `oskar.wi...@gmail.com` on 12 Dec 2010 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
Would the temporary effects include things like adding poison to your weapons, 
or some sort of hardening agent to your armor.  Granted there can also be 
permanent enhancements to equipment too but that is a different point.
```

Original comment by `sdeva...@gmail.com` on 23 Dec 2010 at 2:52

**Comment by GoogleCodeExporter** [comment]
```
ahh this would be cool u could find items or pay i high price to get ur items 
socketed with gems or enchanted with extra health or damage.

```

Original comment by `breid3...@hotmail.com` on 11 Jan 2011 at 2:25

**Comment by GoogleCodeExporter** [comment]
```
Poison - -1 damage dealt per poison counter

Venom - -1 damage dealt per counter, 1 HP per counter per tile moved

Curse - healing is half as effective

Burns - +1 damage recieved per attack landed on you per Burn Counter

Sprain - Move cost +1 AP, attack cost +1 AP and 1HP perswing

Deep Wound - -20% max HP per Wound

Blind - -45% to-hit

Dazed - walk 1 tile random direction every 3 steps until rested

Muddled - cant use spells or skills until rested

Terror - cant attack unless attacked, cant step with 1 tile of enemies


```

Original comment by `rqpa...@gmail.com` on 4 Feb 2011 at 8:08

**Comment by GoogleCodeExporter** [comment]
```
Need to get details figured out. As far as the UI goes there is an area below 
your XP bar that is not being used. Colored text with a countdown could be 
used. The countdown would count how many turns the effect would last. A turn 
can be either an attack or moving.

Example attached.

```

Original comment by `sdeva...@gmail.com` on 10 Feb 2011 at 12:32
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

Attachments:
- [world_effect_UI.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-62/comment-5/world_effect_UI.png)

**Comment by GoogleCodeExporter** [comment]
```
Also, temporary effect should include positive things such as regeneration 
(+1HP per unit of time).

I'm thinking we introduce some definition like a "round" or "turn", maybe 20 
seconds or something like that. Every round, the effect could be applied. So 
for example, being poisoned will lower your HP by 1 each 20 seconds until N 
rounds have passed.

We would also have to indicate type of temporary effect and duration on the 
monsters, items and dialogue options that induce the effects on the player.
```

Original comment by `oskar.wi...@gmail.com` on 10 Feb 2011 at 7:23
- Changed title: **Implement temporary effects, for example poison**

**Comment by GoogleCodeExporter** [comment]
```
This has started for 0.6.9
```

Original comment by `sdeva...@gmail.com` on 3 Mar 2011 at 5:50
- Changed state: **Started**
- Added labels: **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
```
This is now fully implemented in the editors and the game.

Now we only need to create monsters and items that use these new effects :)
```

Original comment by `oskar.wi...@gmail.com` on 14 Apr 2011 at 8:41
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:50
- Changed state: **Done**
